src/flick/requirements.txt

```diff
@@ -21,6 +21,7 @@ click==6.7
 colorama==0.3.9
 configparser==4.0.2
 crayons==0.3.0
+cryptography==3.1
```

Comment:
are you sure you need cryptography? i dont see it imported :o hahaha i might be wrong

Reply:
it is an optional dependency for pyjwt (not needed for some algorithms) for algorithm “ES256” it is needed